### PR TITLE
[yang]Update port yang model, change MTU range to [68,9216]

### PR DESCRIPTION
**Pull request by pettershao-ragilenetworks**
What I did
Filter port invalid MTU configuration

How I did it
Adjust the MTU value to the range of [68,9216]

How to verify it
Use "config interface mtu Ethernet1 40" command to configure the port MTU. The following error will occur in SWSS.

```
Sep 16 14:53:39.051202 sonic NOTICE swss#orchagent: :- doPortTask: Set port Ethernet1 MTU to 40
Sep 16 14:53:39.052217 sonic INFO swss#/supervisord: portmgrd Error: mtu less than device minimum.
Sep 16 14:53:39.052600 sonic ERR swss#portmgrd: :- main: Runtime error: /sbin/ip link set dev "Ethernet1" mtu "40" : 
Sep 16 14:53:39.325117 sonic INFO swss#supervisord 2022-09-16 06:53:39,324 INFO exited: portmgrd (exit status 255; not expected)
Sep 16 14:53:40.339256 sonic INFO swss#/supervisor-proc-exit-listener: Process 'portmgrd' exited unexpectedly. Terminating supervisor 'swss'
Sep 16 14:53:40.352537 sonic INFO swss#supervisord 2022-09-16 06:53:40,339 WARN received SIGTERM indicating exit request
```

We use the iproute2 tool to set the kernel MTU in portmgrd. The command is:
```
ip link set dev <port_name> mtu <mtu>
```

I have tested that iproute2 tool will not report an error when the MTU is in the 68-9216 range.
```
root@sonic:/home/admin# ip link set dev Ethernet10 mtu 68
root@sonic:/home/admin# ifconfig Ethernet10          
Ethernet10: flags=4099<UP,BROADCAST,MULTICAST>  mtu 68
        inet 10.0.0.18  netmask 255.255.255.254  broadcast 0.0.0.0
        ether 58:69:6c:fb:21:38  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

root@sonic:/home/admin# ip link set dev Ethernet10 mtu 67
Error: mtu less than device minimum.

root@sonic:/home/admin# ip link set dev Ethernet10 mtu 9216
root@sonic:/home/admin# ifconfig Ethernet10            
Ethernet10: flags=4099<UP,BROADCAST,MULTICAST>  mtu 9216
        inet 10.0.0.18  netmask 255.255.255.254  broadcast 0.0.0.0
        ether 58:69:6c:fb:21:38  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


root@sonic:/home/admin# ip link set dev Ethernet10 mtu 9217
RTNETLINK answers: Invalid argument
root@sonic:/home/admin#
```

I have changed CLI MTU range in the following PR. 
https://github.com/sonic-net/sonic-utilities/pull/2378

### CODE REVIEW

**Comment by pettershao-ragilenetworks** [comment]
Hi~ could you please review, thanks. @qiluo-msft @dgsudharsan

**Review by dgsudharsan — APPROVED** [review]

File: src/sonic-yang-models/yang-models/sonic-port.yang [hunk 1]
```diff
@@ -117,7 +117,7 @@ module sonic-port{
 					
 				leaf mtu {
 					type uint16 {
-						range 1..9216;
+						range 68..9216;
```
Comment (ganglyu) [inline src/sonic-yang-models/yang-models/sonic-port.yang:120]: Please update unit test to cover this change.

Comment (pettershao-ragilenetworks) [reply src/sonic-yang-models/yang-models/sonic-port.yang:120]: I have updated unit test and comment. Please review. @ganglyu

Comment (ganglyu) [reply src/sonic-yang-models/yang-models/sonic-port.yang:120]: unit test is updated, but where is the comment?

Comment (pettershao-ragilenetworks) [reply src/sonic-yang-models/yang-models/sonic-port.yang:120]: I added it at the beginning.
We use the iproute2 tool to set the kernel MTU in portmgrd. If MTU value is less than 68, iproute2 tool will report an error.

**Review by ganglyu — APPROVED** [review]

**Comment by pettershao-ragilenetworks** [comment]
Please review and merge, thanks. @qiluo-msft

**Pull request merged**